{{cookiecutter.project_slug}}/app/domain/background_task.py

```
@@ -0,0 +1,89 @@
+)
+from django.utils.timezone import now
+from app.models import (
+    CELERY_STATE_MAP,
+    BackgroundTask,
+    BackgroundTaskEvent,
+    BackgroundTaskFile,
+    EventType,
+    TaskStatus,
+)
+
+def attach_file_to_task(task_id, file_obj, description=""):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)